style.css

```diff
@@ -92,7 +118,232 @@
   display: flex;
 }
 
-@media only screen and (max-width: 650px) {
+.btn {
+  color: #6070ff;
+  text-decoration: none;
+  border-radius: 7px;
+  padding: 0.6rem;
+  border: 1px solid #6070ff;
+  font-size: 16px;
+  font-weight: bold;
+}
+
+.btn:hover {
+  color: #fff;
+  background-color: #6070ff;
+  transition: 1s;
+}
+
+.mt-4 {
+  height: 4rem;
+}
+
+.mt-3 {
+  height: 3rem;
+}
+
+.content h5 {
+  text-transform: uppercase;
+  color: #6070ff;
+  font-size: 18px;
+  margin-top: 2.5rem;
+}
+
+.experience {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  width: 60%;
+  justify-items: center;
+  gap: 5px;
+}
+
+.lan-fra {
+  text-align: center;
+}
+
+.expitem {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: 255px;
+  height: 255px;
+  border-radius: 0.7rem;
+  box-shadow: 0 0 3px rgba(141, 141, 141, 0.774);
+  padding: 0.7rem;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 0.7rem;
+  margin-top: 1rem;
+}
+
+.grid-item {
+  height: 100px;
+  width: 100px;
+  background-color: rgb(247, 247, 247);
+  border: #d4d4d4 1px solid;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 1rem;
+  padding: 1.5rem;
+}
+
+.grid-item:hover {
+  background-color: rgb(240, 240, 240);
+  transition: 1s;
+}
+
+.description p {
+  color: grey;
+}
+
+.expitem h3 p {
+  color: #172b4d;
+}
+
+.innerworkcontainer {
+  display: flex;
+  justify-content: center;
+}
+
+.work {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  width: 85%;
+  border-radius: 1.5rem;
+  padding: 3rem;
+}
+
+.aboutme {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  border-top-right-radius: 2.5rem;
+  padding: 1.5rem 5rem 1rem 5rem;
+}
+
+.description {
+  width: 47%;
+  display: flex;
+  flex-direction: column;
+}
+
+.description h1 {
+  font-size: 35px;
+  color: #172b4d;
+}
+
+.company {
+  display: flex;
+  margin-block-start: unset;
+  padding-inline-start: unset;
+}
+
+.company .item1 {
+  color: #172b4d;
+  list-style: none;
+  margin-block-start: unset;
+  margin-block-end: unset;
+}
+
+.company .item2,
+.item3 {
+  color: #6d7583;
+  margin-left: 2rem;
+  margin-block-start: unset;
+  margin-block-end: unset;
+}
+
+.tools {
+  display: flex;
+  margin-block-start: unset;
+  margin-block-end: unset;
+}
+
+.tools .item {
+  padding: 0.5rem;
+  border-radius: 0.7rem;
```

Comment:
- [ ] Please stick to the original design remove the curves from the icons languages